entrypoint.sh

```diff
@@ -80,7 +80,7 @@ fi
 
 # post deploy cache clear
 if [ "${INPUT_CACHE_CLEAR^^}" == "TRUE" ]; then
-    CACHE_CLEAR="&& wp --skip-plugins --skip-themes page-cache flush"
+    CACHE_CLEAR="&& wp --skip-plugins --skip-themes page-cache flush && wp cdn-cache flush"
```

Comment:
We might want to add the `--skip-plugins --skip-themes` flags to the cdn-cache flush for the same reasons as #29.